docker/edge/http/script.sh

```diff
@@ -45,41 +43,46 @@ do
    fi
 done
 
+#cat generate_token.txt
+
 token=`grep "X-Subject-Token" generate_token.txt | cut -f 2 -d ":" |  sed 's/\r$//g' | tr -d '\n'`
 echo -----------------------------
 echo IDM token is $token 
 echo -----------------------------
 
-#Register application, it will return Application ID and Password. ID and Password will use to genenrate access token
+#Fetch Application ID and Password. ID and Password will use to genenrate access token
```

Comment:
[Minor]Fetch application ID and shared secret to authenticate the token shared by IOT devices.

Reply:
Added the said comment in script